lib/renderer/api/remote.js

```diff
@@ -161,10 +161,17 @@ const proxyFunctionProperties = function (remoteMemberFunction, metaId, name) {
     if (loaded) return
     loaded = true
     const meta = ipcRenderer.sendSync('ELECTRON_BROWSER_MEMBER_GET', metaId, name)
-    setObjectMembers(remoteMemberFunction, remoteMemberFunction, meta.id, meta.members)
+    if (Array.isArray(meta.members)) {
+      setObjectMembers(remoteMemberFunction, remoteMemberFunction, meta.id, meta.members)
+    }
   }
 
   return new Proxy(remoteMemberFunction, {
+    set: function(target, property, value, receiver) {
+      if (property !== 'ref') loadRemoteProperties()
```

Comment:
This prevents eagerly loading the function properties when the function is accessed or invoked.